### ISSUE TITLE
[alpha] Install issues

### ISSUE THREAD

**Issue by lxcodes** [Tonkpils/ember-sweetalert#14]
@lindyhopchris I moved this here to not clog up the 1.0 ( #13 ) pull request. 

SweetAlert was still in the bower.json so I removed it from there.  After I cleared out the tmp folders, I'm still getting the same error when the `{{sweet-alert}}` component is triggered with `true` from the `show` computed property.

Template snippet
```
{{sweet-alert
  show=temperatureTooHigh
  title="Temperature Too High"
  text="The temperature entered exceeds the maximum temperature limit."
  icon="error"
  confirmButtonText="Try again"
}}
```

```
  temperatureTooHigh: computed('pumpApplication.temperature', function() {
    return this.get('pumpApplication.temperature') > 300;
  }),
```

ember -v
```
ember-cli: 3.2.0
node: 6.14.1
os: darwin x64
```

package.json
```
{
  "name": "tester",
  "version": "0.0.0",
  "description": "",
  "license": "All Rights Reserved",
  "author": "fh group",
  "directories": {
    "doc": "doc",
    "test": "tests"
  },
  "repository": "",
  "scripts": {
    "build": "ember build",
    "lint:js": "eslint .",
    "start": "ember serve",
    "test": "ember test"
  },
  "devDependencies": {
    "broccoli-asset-rev": "^2.7.0",
    "ember-ajax": "^3.0.0",
    "ember-cli": "~3.2.0",
    "ember-cli-app-version": "^3.0.0",
    "ember-cli-babel": "^6.6.0",
    "ember-cli-bootstrap-sassy": "^0.5.8",
    "ember-cli-dependency-checker": "^2.0.0",
    "ember-cli-eslint": "^4.2.1",
    "ember-cli-htmlbars": "^2.0.1",
    "ember-cli-htmlbars-inline-precompile": "^1.0.0",
    "ember-cli-inject-live-reload": "^1.4.1",
    "ember-cli-moment-shim": "^3.5.0",
    "ember-cli-nouislider": "^0.14.1",
    "ember-cli-qunit": "^4.3.2",
    "ember-cli-sass": "^7.2.0",
    "ember-cli-shims": "^1.2.0",
    "ember-cli-sri": "^2.1.0",
    "ember-cli-uglify": "^2.0.0",
    "ember-cli-update": "^0.21.4",
    "ember-concurrency": "^0.8.16",
    "ember-cpm": "^3.0.0",
    "ember-d3": "^0.4.4",
    "ember-data": "~3.2.0",
    "ember-export-application-global": "^2.0.0",
    "ember-feature-flags": "^4.2.0",
    "ember-file-upload": "^2.5.0",
    "ember-font-awesome": "^3.1.1",
    "ember-get-config": "^0.2.4",
    "ember-inflector": "2.1.0",
    "ember-inputmask": "^0.6.0",
    "ember-load-initializers": "^1.1.0",
    "ember-local-storage": "^1.3.6",
    "ember-lodash": "~4.17.6",
    "ember-maybe-import-regenerator": "^0.1.6",
    "ember-modal-dialog": "^2.4.1",
    "ember-moment": "^7.6.0",
    "ember-page-title": "^4.0.3",
    "ember-parachute": "^0.3.7",
    "ember-resolver": "^4.0.0",
    "ember-select": "0.4.0",
    "ember-simple-auth": "^1.6.0",
    "ember-source": "~3.2.0",
    "ember-sweetalert": "https://github.com/Tonkpils/ember-sweetalert.git#1.0",
    "ember-tri-state": "^1.0.1",
    "ember-truth-helpers": "^2.0.0",
    "ember-watson": "^0.9.1",
    "ember-welcome-page": "^3.0.0",
    "ember-wormhole": "^0.5.4",
    "eslint-plugin-ember": "^5.0.0",
    "loader.js": "^4.2.3",
    "qunit-dom": "^0.6.2"
  },
  "private": true,
  "dependencies": {
    "accounting": "^0.4.1",
    "d3": "^5.3.0",
    "ember-concurrency": "^0.8.16",
    "spinkit": "^1.2.5"
  },
  "engines": {
    "node": "6.* || 8.* || >= 10.*"
  }
}
```
bower.json
```
{
  "name": "tester",
  "dependencies": {
    "canvas": "http://cdnjs.cloudflare.com/ajax/libs/canvasjs/1.4.1/canvas.min.js",
    "jquery.inputmask": "3.1.49",
    "js-quantities": "~1.5.0",
    "print-area": "~2.4.1",
    "ravenjs": "http://cdn.ravenjs.com/3.12.1/raven.min.js",
    "round": "https://gist.githubusercontent.com/al3x-edge/01c8b37ce44476ba8aaa/raw/eb3020675f163f7def10939f71d7f4b94d75b3e1/round.js"
  }
}
```

**Comment by al3x-edge** [comment]
This was the issue for us: If sourcemaps aren't enabled, sweetalert doesn't load.

```
    sourcemaps: {
      // enabled: IS_PROD_LIKE, // CMD ALT F in chrome
      extensions: ['js']
    },
```

**Comment by lindyhopchris** [comment]
Reopening this as it's a bug that I've also encountered.

On the `1.0` branch, if sourcemaps are disabled then sweet alert doesn't load. No idea where to start with this for the moment but need to work out how to get it working without sourcemaps.

**Comment by lxcodes** [comment]
@lindyhopchris Let me know if you need a tester to verify.

**Comment by lindyhopchris** [comment]
Thanks, will do!

**Comment by lindyhopchris** [comment]
Fixed via #18
